libraries/AP_Winch/AP_Winch_Backend.cpp

```diff
@@ -0,0 +1,68 @@
+#include <AP_Winch/AP_Winch_Backend.h>
+#include <RC_Channel/RC_Channel.h>
+
+// setup rc input and output
+void AP_Winch_Backend::init_input_and_output()
+{
+
+    // set servo output range
+    SRV_Channels::set_angle(SRV_Channel::k_winch,  1000);
+
+    RC_Channel *rc_input = rc().find_channel_for_option(RC_Channel::AUX_FUNC::WINCH_CONTROL);
+    if (rc_input != nullptr) {
+        // rc input deadzone is required or pilot input will always override autonomous control of winch
+        rc_input->set_default_dead_zone(30);
```

Comment:
Perhaps make this a prearm check rather than specifying a deadzone.  Setting here will preclude the pilot specifying a wider deadzone.  That will require the user to change another setting, which is a pain.  Middle-ground would be to use a value if it has been set (i.e. is in storage).

Reply:
I think the middle ground is what has been done here.  Note that it's setting the default deadzone.  So if the user has overridden the parameter it will take effect.  I agree the pre-arm is a good idea but I'm going to try and "handle it in documentation" for now.  I.e. the wiki will tell the user to check the deadzone is 30 (which it should default to).  If enough users get themselves into trouble then I'll add the pre-arm check in a follow-up PR.